src/views/portal/VerticalTimeline.tsx

```diff
@@ -0,0 +1,62 @@
+import CheckCircle from "assets/icons/check-circle-fill.svg";
+import XCircle from "assets/icons/x-circle-fill.svg";
+import Image from "next/image";
+import ListGroup from "react-bootstrap/ListGroup";
+import styles from "views/portal/Portal.module.scss";
+
+export interface TimelineProps {
```

Comment:
Question: is this interface used anywhere else so that it needs to be exported?
Nitpick: To follow convention throughout our components, we should name the props interface based on the component name, i.e. `VerticalTimelineProps`.